diff --git a/node_modules/.bin/browser-sync b/node_modules/.bin/browser-sync
deleted file mode 100644
index 678c6e4..0000000
--- a/node_modules/.bin/browser-sync
+++ /dev/null
@@ -1,15 +0,0 @@
-#!/bin/sh
-basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
-
-case `uname` in
-    *CYGWIN*) basedir=`cygpath -w "$basedir"`;;
-esac
-
-if [ -x "$basedir/node" ]; then
-  "$basedir/node"  "$basedir/../browser-sync/bin/browser-sync.js" "$@"
-  ret=$?
-else 
-  node  "$basedir/../browser-sync/bin/browser-sync.js" "$@"
-  ret=$?
-fi
-exit $ret
diff --git a/node_modules/.bin/browser-sync.cmd b/node_modules/.bin/browser-sync.cmd
deleted file mode 100644
index 1f91936..0000000
--- a/node_modules/.bin/browser-sync.cmd
+++ /dev/null
@@ -1,7 +0,0 @@
-@IF EXIST "%~dp0\node.exe" (
-  "%~dp0\node.exe"  "%~dp0\..\browser-sync\bin\browser-sync.js" %*
-) ELSE (
-  @SETLOCAL
-  @SET PATHEXT=%PATHEXT:;.JS;=;%
-  node  "%~dp0\..\browser-sync\bin\browser-sync.js" %*
-)
\ No newline at end of file
diff --git a/node_modules/.bin/browserslist b/node_modules/.bin/browserslist
deleted file mode 100644
index 1df3f2a..0000000
--- a/node_modules/.bin/browserslist
+++ /dev/null
@@ -1,15 +0,0 @@
-#!/bin/sh
-basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
-
-case `uname` in
-    *CYGWIN*) basedir=`cygpath -w "$basedir"`;;
-esac
-
-if [ -x "$basedir/node" ]; then
-  "$basedir/node"  "$basedir/../browserslist/cli.js" "$@"
-  ret=$?
-else 
-  node  "$basedir/../browserslist/cli.js" "$@"
-  ret=$?
-fi
-exit $ret
diff --git a/node_modules/.bin/browserslist.cmd b/node_modules/.bin/browserslist.cmd
deleted file mode 100644
index e4006d7..0000000
--- a/node_modules/.bin/browserslist.cmd
+++ /dev/null
@@ -1,7 +0,0 @@
-@IF EXIST "%~dp0\node.exe" (
-  "%~dp0\node.exe"  "%~dp0\..\browserslist\cli.js" %*
-) ELSE (
-  @SETLOCAL
-  @SET PATHEXT=%PATHEXT:;.JS;=;%
-  node  "%~dp0\..\browserslist\cli.js" %*
-)
\ No newline at end of file
diff --git a/node_modules/.bin/dev-ip b/node_modules/.bin/dev-ip
deleted file mode 100644
index 7133d42..0000000
--- a/node_modules/.bin/dev-ip
+++ /dev/null
@@ -1,15 +0,0 @@
-#!/bin/sh
-basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
-
-case `uname` in
-    *CYGWIN*) basedir=`cygpath -w "$basedir"`;;
-esac
-
-if [ -x "$basedir/node" ]; then
-  "$basedir/node"  "$basedir/../dev-ip/lib/dev-ip.js" "$@"
-  ret=$?
-else 
-  node  "$basedir/../dev-ip/lib/dev-ip.js" "$@"
-  ret=$?
-fi
-exit $ret
diff --git a/node_modules/.bin/dev-ip.cmd b/node_modules/.bin/dev-ip.cmd
deleted file mode 100644
index 0ed7845..0000000
--- a/node_modules/.bin/dev-ip.cmd
+++ /dev/null
@@ -1,7 +0,0 @@
-@IF EXIST "%~dp0\node.exe" (
-  "%~dp0\node.exe"  "%~dp0\..\dev-ip\lib\dev-ip.js" %*
-) ELSE (
-  @SETLOCAL
-  @SET PATHEXT=%PATHEXT:;.JS;=;%
-  node  "%~dp0\..\dev-ip\lib\dev-ip.js" %*
-)
\ No newline at end of file
diff --git a/node_modules/.bin/express b/node_modules/.bin/express
deleted file mode 100644
index ebacee7..0000000
--- a/node_modules/.bin/express
+++ /dev/null
@@ -1,15 +0,0 @@
-#!/bin/sh
-basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
-
-case `uname` in
-    *CYGWIN*) basedir=`cygpath -w "$basedir"`;;
-esac
-
-if [ -x "$basedir/node" ]; then
-  "$basedir/node"  "$basedir/../express/bin/express" "$@"
-  ret=$?
-else 
-  node  "$basedir/../express/bin/express" "$@"
-  ret=$?
-fi
-exit $ret
diff --git a/node_modules/.bin/express.cmd b/node_modules/.bin/express.cmd
deleted file mode 100644
index a799047..0000000
--- a/node_modules/.bin/express.cmd
+++ /dev/null
@@ -1,7 +0,0 @@
-@IF EXIST "%~dp0\node.exe" (
-  "%~dp0\node.exe"  "%~dp0\..\express\bin\express" %*
-) ELSE (
-  @SETLOCAL
-  @SET PATHEXT=%PATHEXT:;.JS;=;%
-  node  "%~dp0\..\express\bin\express" %*
-)
\ No newline at end of file
